.gitignore

```diff
@@ -1,9 +1,9 @@
-.php_cs.cache
-
-# composer related
+/build
+/Resources/doc/_build
+/vendor
 composer.lock
-composer.phar
-vendor
+phpunit.xml
+.php_cs.cache
```

Comment:
Please sort the files as well like in https://github.com/sonata-project/SonataUserBundle/blob/3.x/.gitignore